include/sdf/Element.hh

```diff
@@ -479,6 +491,28 @@ namespace sdf
     // The possible child elements
     public: ElementPtr_V elementDescriptions;
 
+    /// \brief The <include> element that was used to load this entity. For
+    /// example, given the following SDFormat:
+    /// <sdf version = '1.8'>
+    ///   <world name='default'>
+    ///     <include>
+    ///       <uri>model_uri</uri>
+    ///       <pose>1 2 3 0 0 0</pose>
+    ///     </include>
+    ///   </world>
+    /// </sdf>
+    /// The ElementPtr associated with the model loaded from 'model_uri' will
```

Comment:
nit Consider surrounding actual code elements with backticks, e.g. `` `module_uri` ``.

Doxygen supports its own flavor of Markdown.

Reply:
Done in https://github.com/osrf/sdformat/pull/509/commits/2631da14dff94ea7c66e2480be626ffd3e33e72e